src/SGB_Command_Palettes.md

```diff
@@ -81,7 +81,7 @@ The palette data is sent by VRAM-Transfer (4 KBytes).
 ```
 
 Each Palette consists of four 16-bit color definitions (8 bytes). Note:
-The data is stored at 3000h-3FFFh in SNES memory.
+The data is stored at 3000-3FFF in SNES memory.
```

Comment:
```suggestion
The data is stored at $3000-3FFF in SNES memory.
```

Reply:
Same here; I omitted the sigil since it's an address. Most addresses don't have sigils. But if you want it there, shouldn't it be $3000-$3FFF like for other similar ranges?